schema.graphql

```diff
@@ -152,3 +154,17 @@ type Transaction @entity {
   earnings: BigDecimal
   gysrSpent: BigDecimal
 }
+
+type Funding @entity {
+  # geyser id, created at timestamp
+  id: ID!
+
+  geyser: Geyser!
+
+  createdTimestamp: BigInt!
+  start: BigInt!
+  end: BigInt!
+
+  sharesPerSecond: BigDecimal!
+  amount: BigDecimal!
```

Comment:
probably want to store the `shares` value here instead since that is immutable. The original `amount` can change with elastic tokens

Reply:
Great catch. Now using shares instead